### ISSUE TITLE
batch norm updates

### ISSUE THREAD

**Issue by edgarriba** [IshmaelBelghazi/ALI#13]
Hi, any hint about how does batch norm update and why you do the following?
```python
bn_updates = [(p, m * 0.05 + p * 0.95) for p, m in pop_updates]
```

Just trying to mimic the same behavior in Pytorch but not sure what are `p` and `m` and when this update happens during the execution.

**Comment by IshmaelBelghazi** [comment]
This line estimate the population statistics during training using a moving average. `p` and `m` stand for the parameters before and after the update respectively.
